package.json

```diff
@@ -1,20 +1,28 @@
 {
-  "name": "nuxtshop",
-  "private": true,
-  "license": "MIT",
-  "engines": {
-    "node": "^14.16.0 || ^16.11.0 || ^17.0.0"
-  },
   "scripts": {
+    "layer0:dev": "layer0 dev",
+    "layer0:build": "layer0 build",
+    "layer0:deploy": "layer0 deploy",
     "dev": "nuxi dev",
     "build": "nuxi build",
     "start": "node .output/server/index.mjs",
     "get-shopify-fragment-types": "node ./apollo/schemaQuery.mjs",
     "deploy": "layer0 deploy"
   },
+  "name": "nuxtshop",
+  "private": true,
+  "license": "MIT",
+  "engines": {
+    "node": "^14.16.0 || ^16.11.0 || ^17.0.0"
```

Comment:
@adebiyial Does layer0 work with node16 yet? If not we should add something to check

Reply:
Updated.